package.json

```diff
@@ -35,6 +35,9 @@
     "typescript": "^3.6.4"
   },
   "dependencies": {
+    "@iota/core": "^1.0.0-beta.23",
```

Comment:
Are all three of these libraries required just to encode & decode addresses?

Reply:
Yes. They are dependent.

Reply:
You're not directly importing @iota/core, though - if you leave it out and it's required by @iota/validators, it should be picked up automatically.

Reply:
Done.

Reply:
Check please.